opm/autodiff/SimulatorFullyImplicitBlackoilEbos.hpp

```diff
@@ -149,6 +149,29 @@ class SimulatorFullyImplicitBlackoilEbos
             // This is a restart, populate WellState and ReservoirState state objects from restart file
             output_writer_.initFromRestartFile(props_.phaseUsage(), grid(), state, prev_well_state, extra);
             initHydroCarbonState(state, props_.phaseUsage(), Opm::UgGridHelpers::numCells(grid()), has_disgas_, has_vapoil_);
+            {
```

Comment:
This section would be good to have in its own method, to avoid run() getting even longer.